lib/utils/utf8.c

```diff
@@ -5,14 +5,21 @@
  */
 
 #include <stdint.h>
+#undef _POSIX_C_SOURCE
+#define _POSIX_C_SOURCE 200809L
```

Comment:
you need to add it before any header inclusion :)

Reply:
I've made changes :)